app/components/sbom/component-details/vulnerabilities/vulnerability-id/index.hbs

```diff
@@ -0,0 +1,15 @@
+<AkStack @alignItems='center' @spacing='0.5'>
+  <AkIconButton
+    data-test-sbomComponentVulnerabilities-toggleDetailsBtn
+    {{on 'click' @collapsibleContentHanlder}}
```

Comment:
Negligible also but the spelling should be `collapsibleContentHandler`.

Reply:
done